backend/README.md

```diff
@@ -97,22 +97,4 @@ Afterwards, you can quickly re-run tests without running the full tox process:
 
 ```bash
 (ventserver) $ poetry run coverage run --parallel -m pytest --basetemp=.tox/py37/tmp --verbose tests
-```
-
-## Documentation
```

Comment:
If the documentation for auto-docs-generation has now been moved into `backend/docs/README.md`, can you leave a section in `backend/README.md` which points people to `backend/docs/README.md` for instructions about auto-docs-generation?